airflow-pipeline/README.MD

```diff
@@ -0,0 +1,79 @@
+## Init project
+
+1.1 Install gradle
+    
+    https://docs.gradle.org/current/userguide/installation.html
+    
+1.2 Init gradle project
+
+
+    gradle init
+
+    
+1.3 Use build.gradle from airflow-pipeline directory. Change mainClassName variable.
+
+
+## Setup airflow project
+
+### Prepage amazon S3
+
+Create your own directory into the common bucket “student-labs” on S3 to persist job artifacts.
+URI to access the directory can look like:
+
+```s3a://student-labs/ivanov```
+
+### Codebase preparation
+
+1. Change patches in the file configuration according to your S3 directory. Notice that path must be started with s3a:// to use S3A committer.
+
+   Add patches to datasets:
+
+   ```s3a://scala-spark-course/dataset/mobile_app_clickstream/mobile_app_clickstream_0.csv.gz```
```

Comment:
You should folder in the path here, not file

Reply:
I've removed those at all. General README already included this information.